website/content/api-docs/secret/ssh.mdx

```diff
@@ -210,8 +210,10 @@ This endpoint creates or updates a named role.
   '{{public_key_hash}}' - A SHA256 checksum of the public key that is being signed.
   e.g. "custom-keyid-{{token_display_name}}"
 
-- `allowed_user_key_lengths` `(map<string|int>: "")` – Specifies a map of ssh key types
+- `allowed_user_key_lengths` `(map<string|int|[]int>: "")` – Specifies a map of ssh key types
```

Comment:
Should this type signature be `map<string|(int|[]int)>` instead perhaps? Not really sure how to indicate an optional list of int-like types.